src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/StorageSection/CustomizeSource/StorageClassSelect/StorageClassSelect.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, MouseEvent, useMemo, useState } from 'react';
+
+import useDefaultStorageClass from '@kubevirt-utils/hooks/useDefaultStorage/useDefaultStorageClass';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { FormGroup, Select, SelectOption } from '@patternfly/react-core';
+
+type StorageClassSelectProps = {
+  onStorageClassChange: (_: MouseEvent, value: string) => void;
+  storageClassName: string;
+  storageClassRequired: boolean;
+};
+
+const StorageClassSelect: FC<StorageClassSelectProps> = ({
+  onStorageClassChange,
+  storageClassName,
+  storageClassRequired,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [{ storageClasses }] = useDefaultStorageClass();
+
+  const [isOpenStorageClass, setIsOpenStorageClass] = useState<boolean>(false);
+
+  const sortedStorageClasses = useMemo(
+    () => storageClasses?.map((sc) => sc?.metadata?.name)?.sort(),
+    [storageClasses],
+  );
```

Comment:
maybe move this into useDefaultStorageClass?

Reply:
Good idea, done thanks :)